tmp/lanelet2_extension/lib/visualization.cpp

```diff
@@ -105,14 +105,15 @@ void initLightMarker(visualization_msgs::msg::Marker * marker, const std::string
   marker->scale.z = s;
 }
 
-bool inputLightMarker(visualization_msgs::msg::Marker * marker, const lanelet::ConstPoint3d & p)
+bool inputLightMarker(
+  visualization_msgs::msg::Marker * marker, const lanelet::ConstPoint3d & p, const int & id)
 {
   if (marker == nullptr) {
     std::cerr << __FUNCTION__ << ": marker is null pointer!" << std::endl;
     return false;
   }
 
-  marker->id = static_cast<int32_t>(p.id());
```

Comment:
Could you explain a bit more why we need to set ID externally?
The original code uses the ID of a Lanelet point used to define light bulbs of traffic light, and they should have unique IDs if it is a valid Lanelet.